box.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+class Box:
+    def __init__(self, box_dim):
+        self.box_dim = box_dim
+        
+    def wrap(self, coordinates):
+        """Wraps the coordinates within the box dimensions
+
+        Parameters
+        ----------
+        box_dim : np.array
+            Dimensions of the box in reduced units.
+        coordinates : np.array
+            Array of the atomic coordinates.
+
+        Returns
+        -------
+        coordinates : np.array
+            Arrays of the wrapped atomic coordinates.
+        """
+        if len(coordinates.shape) == 1:
```

Comment:
I think it should be if ' len( coordinates.shape[0] ) == 1 : ',  cuase coordinates.shape is always a tuple of len = 2, i.e. (n_particles, 3)

Reply:
We are trying to support passing in one particle's coordinates or a number of particles. If you pass one particle position the shape would be `(3,)` so `len(coordinates.shape)=1`.

Reply:
got it! altough it will depend on the way Particles class creates and manage the coordinates, right?

Reply:
Not necessarily for instance if passed `particles.coordinates[0]` the shape would be `(3,)` but
`particles.coordinates[:10]` would be `(10, 3)`.